torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -208,6 +208,16 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
+        "prim::is_cuda(Tensor a) -> bool",
```

Comment:
nit: Maybe it's better for this to be in the aten:: namespace? See https://github.com/pytorch/pytorch/blob/3ff70712c29f7bc61f3861cae1ce6ce3122a30b2/tools/jit/templates/register_aten_ops.cpp#L62-L84. It doesn't really matter either way

Reply:
See comment in compiler.cpp:
// functions that are just direct property lookups on tensor
// must be registered as prim::<name>(Tensor t) -> <return_type>